interceptor/opencensus/trace_interceptor.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocinterceptor
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"sync"
+
+	"google.golang.org/grpc"
+
+	agenttracepb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/trace/v1"
+	"github.com/census-instrumentation/opencensus-service/interceptor"
+	"github.com/census-instrumentation/opencensus-service/spanreceiver"
+)
+
+// New will create a handle that runs an OCInterceptor at the provided address
+func NewTraceInterceptor(port int, opts ...OCOption) (interceptor.TraceInterceptor, error) {
```

Comment:
Not really, because the port is the only mandatory argument when starting an OpenCensus trace interceptor, while all the others are options. Below is a constructor to use the default port.

Reply:
If it has a default value and we have a version without it, it means for me that it is optional :)